portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -49,7 +53,7 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     PreparedQuery results = datastore.prepare(query);
 
     List<Comment> comments = new ArrayList<>();
-    for (Entity entity : results.asIterable()) {
+    for (Entity entity : results.asIterable(FetchOptions.Builder.withLimit(MAX_COMMENTS_NUM))) {
```

Comment:
I was wondering whether there could be ways to restrict the max_num in query itself but looking at the API, it seems not.

Reply:
I actually changed that later on when I add a query string to the request sent to the servlet.